mappings/net/minecraft/client/item/TooltipContext.mapping

```diff
@@ -1,4 +1,13 @@
 CLASS net/minecraft/unmapped/C_taebrtdw net/minecraft/client/item/TooltipContext
-	METHOD m_ecuysqaa isAdvanced ()Z
+	FIELD f_yceowjjc HIDE_ADVANCED_DETAILS Lnet/minecraft/unmapped/C_taebrtdw$C_vsiqutyg;
+	FIELD f_zuchejxz SHOW_ADVANCED_DETAILS Lnet/minecraft/unmapped/C_taebrtdw$C_vsiqutyg;
+	METHOD m_ecuysqaa shouldShowAdvancedDetails ()Z
+		COMMENT Whether the tooltip should show its advanced details, which are revealed with F3+H.
+	METHOD m_wmqdlnmf shouldShowCreativeDetails ()Z
+		COMMENT Whether the tooltip should show creative-mode-only details.
```

Comment:
oh dear, i was overthinking this javadoc but having this look simple like this? it's helpful!
just one thing:
```suggestion
		COMMENT Whether the tooltip should show details exclusive to the creative inventory.
```
The context is like, it only appears if it's inside the creative inventory's item group; Put it on the hotbar, and yeah, it disappears

Reply:
oh, I was going off memory, thanks for the clarification!